daprdocs/content/en/reference/components-reference/supported-bindings/s3.md

```diff
@@ -169,6 +169,14 @@ To perform a create operation, invoke the AWS S3 binding with a `POST` method an
 }
 ```
 
+Optionally, provide a storage class while using the `create` operation with the following:
+
+```bash
+curl -d '{ "operation": "create", "data": "YOUR_BASE_64_CONTENT", "metadata": { "storageClass": "STANDARD_IA" } }' 
```

Comment:
This is about adding the metadata property into the example scheme (above) and a description in the table (above) for this property. This curl example is not that important, but you can keep this here. However I would then also change the example json above to include a metadata field like this to be consistent with this example. Like this
```json
{
  "operation": "create",
  "data": "YOUR_BASE_64_CONTENT"
  "metadata": {
               "storageClass": "STANDARD_IA" 
    }
}
```